web-app/src/app/pojo/Monitor.ts

```diff
@@ -5,8 +5,8 @@ export class Monitor {
   name!: string;
   app!: string;
   host!: string;
-  intervals: number = 120;
-  // 监控状态 0:未监控,1:可用,2:不可用,3:不可达,4:挂起
+  intervals: number = 60;
+  // 监控状�?0:未监�?1:可用,2:不可�?3:不可�?4:挂起
```

Comment:
```suggestion
  // 监控状态 0:未监控,1:可用,2:不可用,3:不可达,4:挂起
```
Hi please fix encoding error